package.json

```diff
@@ -10,7 +10,8 @@
     "pretest": "npm run lint",
     "test": "tap tests/*.js --100",
     "prerelease": "npm test",
-    "release": "standard-version"
+    "release": "standard-version",
+    "postinstall": "opencollective-postinstall"
```

Comment:
imo it's not appropriate for dotenv to broadcast in this way (also requires dependencies which we've avoided as a selling point in the past)

Reply:
I signed up to explore it.

I agree with you. Let's close this, and we can discuss sometime in the future if it makes sense. GitHub Sponsors today looks interesting as well.